cmd/bootm.c

```diff
@@ -123,7 +123,7 @@ int do_bootm(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 			return do_bootm_subcommand(cmdtp, flag, argc, argv);
 	}
 
-#ifdef CONFIG_IMX_HAB
+#if defined(CONFIG_IMX_HAB) && !defined(CONFIG_IMX8M)
```

Comment:
As I said on the previous PR, can you instead here create a new CONFIG flag that covers this path in a more generic way? Just because the same path will be required by other socs, and we will have to keep extending the definitions here.

A single config can easily be enable on boards that requires it, if needed.

Reply:
And same applies for all the other defined(CONFIG_IMX8M) lines.